zfs/zfs.go

```diff
@@ -1111,6 +1144,17 @@ func ZFSRecv(ctx context.Context, fs string, v *ZFSSendArgVersion, streamCopier
 		}
 		args = append(args, "-s")
 	}
+	if opts.InheritProperties != nil {
+		for _, prop := range opts.InheritProperties {
+			args = append(args, "-x", string(prop))
```

Comment:
FreeBSD 12 does not support `-x` nor `-o` ( https://www.freebsd.org/cgi/man.cgi?zfs(8) )

Thus we'll need feature detection for this, similar to how we do for resumable send ( https://github.com/zrepl/zrepl/blob/44bd354eaedafd7eb5e92e4cf8f7b896e996459a/zfs/resume_token.go#L42 )